### ISSUE TITLE
I have two questions for your paper and code.

### ISSUE THREAD

**Issue by daikitemp** [LiJunnan1992/MLNT#11]
Dear Junnan Li,

I read your paper and it was so impressed. I have two questions for your paper and code.
First of all, what is args.alpha in your code (main.py line 31)? I read your paper, but it seems that it was not written. Could you tell me about this alpha?
Finally, how can I do iterative learning? I could train 1 epoch, but I couldn't do iterative learning 3 epochs like your paper. Could you help me reproduce your paper?

Sorry to ask this of you when you are busy but I appreciate your help.
Thanks so much.

**Comment by LiJunnan1992** [comment]
Hi, thanks for your interest in our paper!

Alpha controls the warm-up weight of the consistent loss in the 1st epoch.

To perform iterative learning, you can run main.py for a second time and load the model from previous iteration as pretrain_net.

**Comment by daikitemp** [comment]
I appreciate your response.

>Alpha controls the warm-up weight of the consistent loss in the 1st epoch.

I was able to understand the content very well.
>To perform iterative learning, you can run main.py for a second time and load the model from previous iteration as pretrain_net.

I understand. Should I add data extraction (D') from training set D with mentor model to it?

Sorry for the numerous questions but I have one more thing to ask. 
I would like to perform on the CIFAR dataset, but the image size of CIFAR is small unlike clothing 1M. Did you increase the image size when experimenting with CIFAR, or did you use another network?

I apologize for the trouble, but please do your best.

**Comment by LiJunnan1992** [comment]
I use a smaller network for CIFAR, you can find it here: https://github.com/LiJunnan1992/DivideMix/blob/master/PreResNet.py.

**Comment by daikitemp** [comment]
Thank you for your prompt response.
>I use a smaller network for CIFAR, you can find it here: https://github.com/LiJunnan1992/DivideMix/blob/master/PreResNet.py.

I will try that.

I am truly sorry for disturbing you during a busy time, but thank you for your cooperation.
Thanks you so much!

**Comment by AllenPu** [comment]
> Thank you for your prompt response.
> 
> > I use a smaller network for CIFAR, you can find it here: https://github.com/LiJunnan1992/DivideMix/blob/master/PreResNet.py.
> 
> I will try that.
> 
> I am truly sorry for disturbing you during a busy time, but thank you for your cooperation.
> Thanks you so much!

Can u use the code for cifar10?
